### ISSUE TITLE
Attempting data recovery

### ISSUE THREAD

**Issue by E3lleyyaa** [libimobiledevice/idevicerestore#691]
```
Got status message
Status: Restore Finished
```
After the firmware update, instead of switching on, the device rebooted and entered a new mode where it shows a different boot screen with the message ‘Attempting data recovery.’ As far as I understand, this process cannot be skipped and installs the firmware a second time, correct?Sorry for image quality(webcam)
![Photo-2](https://github.com/user-attachments/assets/051f925f-e237-46e1-9a56-c92171c50929)

@nikias @doronz88

**Comment by nikias** [comment]
No, this essentially rewrites the encrypted data partition so you don't lose use your data.

**Comment by E3lleyyaa** [comment]
@nikias Is this something new, or has it always been there? It’s the first time I’ve seen it, even though I’ve used it many times. Is there no way to skip it or handle the condition?

**Comment by E3lleyyaa** [comment]
@nikias Thanks for the information! Yes, that's correct—the device was manually switched to recovery mode. I'll try again from normal mode.
